src/main/java/seedu/address/logic/commands/cardcommands/AddCommand.java

```diff
@@ -147,9 +147,6 @@ public Tag getTag() {
          * @return The new Card instance.
          */
         public Card buildCard() {
-            if (tag == null) {
```

Comment:
Bring null check to Card constructor for clarity. If tag is null, will instantiate with UNTAGGED